english/explore-outside/journey-to-flaming-tower/walk/portal/dinner/tell-imp/tell-imp.md

```diff
@@ -0,0 +1,9 @@
+You tell the imp about your concerns. "I don't want to stay this way! I will
+never be able to see my friends again.", I said.  The imp looked sad and
+then compassionate. "I too had friends once," said the imp empathetically. 
+
+You both sit down at his rustic wood table for some herbal tea. Your worries subside a bit, so you tell the imp about your crazy adventure in the marshmallow house.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.